cloud-aws/src/main/resources/definitions/environment-minimal-policy.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
What is the decisive factor for what shall go into the freeipa policy and what into this one? I am slightly confused; we have EC2 here but not Autoscaling. Is this for creating network & DDB tables?

Reply:
This policy contains only vpc/subnet and dynamodb permissions.

Reply:
Super, got it.